requester/requester.go

```diff
@@ -16,17 +16,24 @@ type Requester struct {
 
 	config config.Config
 	client http.Client
+	tracer *tracer
 }
 
 // New returns a Requester configured with specified Options.
 func New(cfg config.Config) *Requester {
+	tracer := newTracer()
 	return &Requester{
 		records: make(chan Record, cfg.RunnerOptions.Requests),
 		config:  cfg,
+		tracer:  tracer,
```

Comment:
Suggestion: inline `newTracer()` here ?

Reply:
I think it wouldn't work: we need the same `tracer` as the one used in `Requester.client.Transport` (L36).

We could do without any `Requester.tracer` however, retrieving the actual tracer it via `Requester.client.Transport.(*tracer)` assertion, but I feel the current approach is cleaner and safer 🤔